src/test/java/ExportTest.java

```diff
@@ -0,0 +1,61 @@
+import org.junit.Test;
+import org.tinylog.Logger;
+import org.variantsync.diffdetective.diff.difftree.DiffTree;
+import org.variantsync.diffdetective.diff.difftree.serialize.*;
+import org.variantsync.diffdetective.diff.difftree.serialize.edgeformat.DefaultEdgeLabelFormat;
+import org.variantsync.diffdetective.diff.difftree.serialize.nodeformat.LabelOnlyDiffNodeFormat;
+import org.variantsync.diffdetective.diff.difftree.serialize.nodeformat.LineNumberFormat;
+import org.variantsync.diffdetective.diff.difftree.serialize.treeformat.CommitDiffDiffTreeLabelFormat;
+import org.variantsync.diffdetective.diff.difftree.serialize.Format;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class ExportTest {
+    /**
+     * Format used for the test export.
+     */
+    private final static Format format =
+        new Format(
+            new LineNumberFormat(),
+            new DefaultEdgeLabelFormat()
+        );
+
+    /**
+     * Format used to deserialize the test case.
+     */
+    private final static DiffTreeLineGraphImportOptions importOptions =
+        new DiffTreeLineGraphImportOptions(
+            GraphFormat.DIFFTREE,
+            new CommitDiffDiffTreeLabelFormat(),
+            new LabelOnlyDiffNodeFormat(),
+            new DefaultEdgeLabelFormat()
+        );
+
+    @Test
+    public void export() throws IOException {
```

Comment:
This test case relies on Graphviz (`dot`) being installed. Using JUnit5 its possible to disable this test if Graphviz is not installed. I will pull request for this later.

Reply:
See #54